python/graphstorm/run/gsgnn_lp/gsgnn_lp.py

```diff
@@ -45,11 +45,43 @@
                                     FastGSgnnLPJointNegDataLoader,
                                     FastGSgnnLPLocalUniformNegDataLoader,
                                     FastGSgnnLPLocalJointNegDataLoader)
-from graphstorm.eval import GSgnnMrrLPEvaluator
+from graphstorm.eval import GSgnnMrrLPEvaluator, GSgnnPerEtypeMrrLPEvaluator
 from graphstorm.model.utils import save_embeddings
 from graphstorm.model import do_full_graph_inference
 from graphstorm.utils import rt_profiler, sys_tracker, setup_device
 
+def get_evaluator(config, train_data):
```

Comment:
is this one the same as above?

Reply:
it doesn't look like this evaluator depends on GNN.

Reply:
It does not.
But as both gsgnn_lp.py and gsgnn_lm_lp.py are main entrances for different training types, I want them to be self contained (without relative include(s)).

Reply:
should we implement a helper function in the core package?

Reply:
Maybe we can implement one for all different tasks.